cmd/install.go

```diff
@@ -88,6 +91,19 @@ To force install a package:
 		// Load config protect configs
 		installer.LoadConfigProtectConfs(LuetCfg)
 
+		// Load finalizer runtime environments
+		if len(finalizerEnvs) > 0 {
```

Comment:
I'm trying slowly bit by bit trying to cleanup the whole `cmd/` code. Can you split this off to its own function here https://github.com/mudler/luet/tree/master/cmd/util ?

Reply:
Ok, n/p.